assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,42 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
+let myFunction = () => {};
+console.log(myFunction);
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
+let anotherFunction = (param) => {
+    return param;
+}
+console.log(anotherFunction);
 
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
+let add = (param1, param2) => {
```

Comment:
same thing, an alternative is:
`let add = (param1, param2) => param1 + param2;`
But yours totally works